specification/application/conventions.tex

```diff
@@ -7,14 +7,34 @@ \section{Application-level conventions}\label{sec:application_level_conventions}
 
 \subsection{Node identifier distribution}
 
-An overview of related concepts is provided in chapter \ref{sec:basic_concepts}.
+An overview of related concepts is provided in chapter~\ref{sec:basic}.
 
 Valid values of node-ID range from 0 up to a transport-specific upper boundary
 which is guaranteed to be above 127 for any transport.
 
 The two uppermost node-ID values are reserved for diagnostic and debugging tools;
 these node-ID values should not be used in fielded systems.
 
+\subsection{Timing constraints}
+
+\subsubsection{Recommended message publication timing constraints}
+
+Generally, it is recommended to abort a message transmission if it cannot be completed in 1 second.
```

Comment:
This part has been carried over from the very first v0 draft. The justification for it being there was that we expected that it is a sensible default for many applications. Given the recent developments, it would be wise to remove any specific numbers from here and limit the section to a more abstract recommendation to have stale data removed from the transmission queue. Would you volunteer to provide exact wording here in a sentence?